app/test_main.py

```diff
@@ -1,3 +1,29 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_human_age_zero() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_human_age_in_1_year() -> None:
+    assert get_human_age(15, 15) == [1, 1]
```

Comment:
This test case seems to be testing the same scenario as the one in line 13. According to the checklist, tests should not repeat the same case multiple times. Consider removing or modifying one of these tests to cover a different scenario.